crates/core/tedge_api/src/health.rs

```diff
@@ -64,7 +64,7 @@ impl ServiceHealthTopic {
                     .with_retain()
             }
             Err(e) => {
-                let error_topic = Topic::new_unchecked("tedge/errors");
+                let error_topic = Topic::new_unchecked(&format!("{topic_root}/errors"));
```

Comment:
It would be better to add a method to `MqttSchema`.

```suggestion
                let error_topic = mqtt_schema.error_topic();
```

Reply:
Added `error_topic` function in mqtt_schema